### DISCUSSION TITLE
Having Difficulty Linking CSS on Github pages

### DISCUSSION THREAD

**Discussion by lgomez92** [community/community#136981] [New to GitHub]
Hi all, 

new user using GitHub, I just deployed a page, but for some reason the CSS is not linking,

Any advice would be appreciated it.

https://github.com/lgomez92/TableCheatSheet

Thanks

**Comment by radar-blip** [comment]
In your index.html, just remove the slash.

Change from this: 
`<link rel="stylesheet" href="/style.css">`

to this:
  `<link rel="stylesheet" href="style.css">`

**Reply by lgomez92** [reply]
> In your index.html, just remove the slash.
> 
> Change from this: `<link rel="stylesheet" href="/style.css">`
> 
> to this: `<link rel="stylesheet" href="style.css">`

Thanks for your reply, I just did that, but unfortunately still not working, I re-deploy the page and still not working.

**Reply by radar-blip** [reply]
Maybe it just took a while. It's working fine now. 

https://lgomez92.github.io/TableCheatSheet/

**Reply by lgomez92** [reply]
Thank you so much, It is working now!

**Reply by radar-blip** [reply]
Anytime. Happy to help!

**Reply by ghostinhershell** [reply]
Hi @lgomez92 👋🏾,It looks like @linuxlawson's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)

**Comment by davevad93** [comment]
Hi @lgomez92 ,

I have checked your site and `style.css` seems to be linked correctly, in the browser dev tools the css doesn't return any errors.

![00000](https://github.com/user-attachments/assets/850ab038-ce18-44a3-8c43-946713b52423)

**Reply by lgomez92** [reply]
Thank you, It is working now!

**Comment by Raykesh-Soneka** [comment]
Hi Luis Gomez (@lgomez92 ),
 
The discussions is still Unanswered? can you change it?
![image](https://github.com/user-attachments/assets/fc560322-0536-4a7b-bd05-8e5c92dd2960)


Thank you
